glyphs2fontir/src/source.rs

```diff
@@ -221,7 +221,7 @@ impl Work for GlyphIrWork {
             let master = &font.font_master[*master_idx];
             let location = &font_info.master_locations[master.id.as_str()];
 
-            for (tag, coord) in location.iter() {
+            for (tag, coord) in location.0.iter() {
```

Comment:
yea I'd prefer that location just get its own little `iter` method that returns `impl Iterator`.

Reply:
Oh good call. I forgot impl Iterator and didn't think returning the specific impl BTreeMap provided resulted in much hiding of implementation. Fixed.